docs/usecases/testing.md

```diff
@@ -48,6 +48,68 @@ In **ZIO Test**, all tests are immutable values and tests are tightly integrated
 
 The library also includes built-in mock versions of all the standard ZIO environmental effects (`Clock`, `Console`, `System`, and `Random`), so when we tested our program above the `helloWorld` method didn't actually print a String to the console but instead wrote the String to a buffer that could access for testing purposes. If you ever do need to access the "live" environment just use the `live` method in the `mock` package or specify the live environment in your type signature like `Live[Console]`.
 
+_Mocking Random_
+
+When working with randomness it is easier to controll what the radnom producer supplies. This can be controlled by a number of methods that allow for filling up buffers with values.
```

Comment:
```suggestion
When working with randomness it is easier to control what the random producer supplies. This can be controlled by a number of methods that allow for filling up buffers with values.
```